### ISSUE TITLE
strange thing about analyze time error

### ISSUE THREAD

**Issue by GoodenoughPhysicsLab** [mido/mido#586]
**Describe the bug**
[bug_report.zip](https://github.com/mido/mido/files/15283198/bug_report.zip)
```Python
# -*- coding: utf-8 -*-
import mido
import copy

SOURCE: str = "example.mid"
OUTPUT: str = "temp.mid"

def test1(): # bug
    mid = mido.MidiFile(SOURCE)
    mid2 = mido.MidiFile()

    for t in mid.tracks:
        mid2.tracks.append(copy.deepcopy(t))

    mid2.save(OUTPUT)


def test2(): # no bug
    mid = mido.MidiFile(SOURCE)
    mid.save(OUTPUT)

test1()
# test2()
```
I expect that temp.mid is a copy of example.mid both test1 and test2
for test1, the speed of output is faster than the origin
however, for test2, the output is normol

I also have another examples, so, this is not a single example

and I also try to change `ticks_per_beat`, but it has no effect.

**Context (please complete the following information):**
- Operating system name & version: win7
- Python version: python 3.8
- mido package version (Retrievable using `mido.version_info`) & installation method (Distribution repository, PyPI, source…): <Version('1.3.2')> installed from PyPI
- backend used (amidi, portmidi, rtmidi, PyGame…): python-rtmidi

**Comment by rdoursenaud** [comment]
Hi and thank you for using mido.

In you first case, you are creating a new `MidiFile` object without passing initial parameters values.
This uses the default values which are different from your source file.
When adding track events, you are copying events with a different resolution which results in playback speed difference.

You need to initialize your destination object with the same settings as your source, especially the resolution.

Please see #576.

```py
[...]
def test():  # The right way
    mid = mido.MidiFile(SOURCE)
    mid2 = mido.MidiFile(
        ticks_per_beat=mid.ticks_per_beat,
        type=mid.type,
        charset=mid.charset,
    )

    for t in mid.tracks:
        mid2.tracks.append(copy.deepcopy(t))

    mid2.save(OUTPUT)
[...]
```

TL;DR: mido behaves as expected here and the file’s resolution (`ticks_per_beat`) is definitely the issue.
